low-level/files/opt/seagate/sspl/setup/sspl_reset.py

```diff
@@ -24,67 +24,32 @@ def del_file(self, filename):
         if os.path.exists(filename):
             os.remove(filename)
 
+    def del_files_from_dir(self, fformat, dir_path):
+        for root, dirs, files in os.walk(dir_path):
+            for file in files:
+                if file.endswith(fformat):
+                    os.remove(os.path.join(root, file))
+
     def process(self):
-        # stop sspl service
         dbus_service = DbusServiceHandler()
-        dbus_service.stop('sspl-ll.service')
-
-        # Remove sspl_conf
-        self.del_file(file_store_config_path)
-
-        # Remove sspl-configured file
-        self.del_file(SSPL_CONFIGURED)
+        # Stop SSPL service if state is active
+        service_state = dbus_service.get_state('sspl-ll.service')
+        if service_state._state == 'active':
+            logger.info(f'For {self.name}, SSPL service should be stopped'
+            'through HA interfaces')
```

Comment:
continuation line under-indented for visual indent. Please make sure to run flake8 before submitting PR.

Reply:
Fixed flake8 issues.